hotel-api-model/src/main/java/com/hotelapp/hotelapp/model/Hotel.java

```diff
@@ -0,0 +1,47 @@
+package com.hotelapp.hotelapp.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Hotel {
```

Comment:
Would equals and hashcode and toString be of any use in this POJO?

Reply:
Good job overriding `equals`, `hashCode` and `toString`